base/sparse/linalg.jl

```diff
@@ -585,39 +585,41 @@ function normestinv{T}(A::SparseMatrixCSC{T}, t::Integer = 2)
         S_old = copy(S)
         for j = 1:t
             for i = 1:n
-                S[i,j] = Y[i,j]==0?1:sign(Y[i,j])
+                S[i,j] = Y[i,j]==0?1:(Y[i,j]/abs(Y[i,j]))
```

Comment:
`sign(z::Complex)` is already defined this way, but is more efficient for non-complex types.